src/style.css

```diff
@@ -1 +1,84 @@
 /* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 14px;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.nav__item {
+
+  list-style: none;
+  padding: 0;
+
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  /* margin: 0 50px; */
+  height: 60px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  display: flex;
+  align-items: center;
+  height: 60px;
+
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  text-align: center;
+  margin-right: 50px;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-left: 50px;
+}
```

Comment:
You are duplicating styles here. Let's create an additional class to avoid repeating.